azure-stack/hci/deploy/deployment-tool-set-up-first-server.md

```diff
@@ -70,40 +70,64 @@ The following procedure provides a typical set of permissions to the custom role
 
     ```jason
     {
-	    "Name": "Azure Stack HCI registration role - Custom",
-	    "Id": null,
-	    "IsCustom": true,
-	    "Description": "Custom Azure role to allow subscription-level access to register Azure Stack HCI",
+	    "id": "/subscriptions/<Azure subscription ID>",
+        "properties": {
+        "roleName": "Azure Stack HCI 23H2 validator and registration role",
+        "description": "Custom Azure role to allow subscription-level access to register Azure Stack HCI",
+        "assignableScopes": [
+            "/subscriptions/<Azure subscription ID>"
+        ],
+        "permissions": [
+        {
 	    "Actions": [
 		"Microsoft.Resources/subscriptions/resourceGroups/read",
-		"Microsoft.Resources/subscriptions/resourceGroups/write",
-		"Microsoft.Resources/subscriptions/resourceGroups/delete",
-		"Microsoft.AzureStackHCI/register/action",
-		"Microsoft.AzureStackHCI/Unregister/Action",
-		"Microsoft.AzureStackHCI/clusters/*",
-		"Microsoft.Authorization/roleAssignments/write",
-		"Microsoft.Authorization/roleAssignments/read",
-		"Microsoft.HybridCompute/register/action",
-		"Microsoft.GuestConfiguration/register/action",
-		"Microsoft.HybridConnectivity/register/action",
-		"Microsoft.HybridCompute/machines/extensions/write",
-		"Microsoft.HybridCompute/machines/extensions/read",
-		"Microsoft.HybridCompute/machines/read",
-		"Microsoft.HybridCompute/machines/write",
-		"Microsoft.HybridCompute/privateLinkScopes/read",
-		"Microsoft.GuestConfiguration/guestConfigurationAssignments/read",
-		"Microsoft.ResourceConnector/register/action",
-		"Microsoft.Kubernetes/register/action",
-		"Microsoft.KubernetesConfiguration/register/action",
-		"Microsoft.ExtendedLocation/register/action",
-		"Microsoft.HybridContainerService/register/action",
-		"Microsoft.ResourceConnector/appliances/write"
+        "Microsoft.Resources/subscriptions/resourceGroups/write",
+        "Microsoft.Resources/subscriptions/resourceGroups/delete",
+        "Microsoft.AzureStackHCI/register/action",
+        "Microsoft.AzureStackHCI/Unregister/Action",
+        "Microsoft.AzureStackHCI/clusters/*",
+        "Microsoft.Authorization/roleAssignments/write",
+        "Microsoft.Authorization/roleAssignments/read",
+        "Microsoft.HybridCompute/register/action",
+        "Microsoft.GuestConfiguration/register/action",
+        "Microsoft.HybridConnectivity/register/action",
+        "Microsoft.HybridCompute/machines/extensions/write",
+        "Microsoft.HybridCompute/machines/extensions/read",
```

Comment:
Please add Microsoft.HybridCompute/machines/extensions/delete permission as well.

Reply:
Added the permission in the code block and the table.